### ISSUE TITLE
Bundle does not work properly with new Intl ICU translations

### ISSUE THREAD

**Issue by bocharsky-bw** [php-translation/symfony-bundle#300]
1. When I extract translations with `$ bin/console translation:extract` - the command put Intl ICU translations to the standard `messages.en.xlf` files instead of `messages+intl-icu.en.xlf`.
2. Even when I have "Intl ICU" translations in `messages+intl-icu.en.xlf` - the `$ bin/console translation:extract` command put those translations into `messages.en.xlf` anyway, i.e. so I have duplications in both files.
3. Xliff format does not support `+` sign in file ID, i.e: `<file id="messages+intl-icu.en">` is invalid ID, at least PhpStorm says so:
<img width="727" alt="Screen Shot 2019-03-20 at 22 50 23" src="https://user-images.githubusercontent.com/3317635/54718176-a1816200-4b62-11e9-8732-b2320f0ed838.png">
4. The Web UI, i.e. `/admin/_trans/app` page does not display translations of `messages+intl-icu.en.xlf` as a separate domain, it displays them in `messages` domain. But if you edit those messages, they popup in messages.en.xlf
5. Probably something else...

**Comment by Guite** [comment]
Point 2 is solved by https://github.com/php-translation/symfony-storage/pull/49 and https://github.com/symfony/symfony/pull/35430

Point 1 is still a huge problem though. It would be great if the extractor could be enforced to append the `+intl-icu` suffix to all domains.

**Comment by tristanbes** [comment]
Just came accross the issue after discovering that translating a plural string has changed starting from Symfony 4.2

The problem is that it will render the string "as is" without the plural form
Outputs: 
> {COUNT, PLURAL, =0 {THERE ARE NO APPLES} ONE {THERE IS ONE APPLE...} OTHER {THERE ARE # APPLES!}}

and I didn't find in the documentation if there is any specific steps that I need to do to achieve what is done in the Symfony documentation (https://symfony.com/doc/4.4/translation/message_format.html#pluralization) 

Steps to reproduce
```xlf
// messages.en.xlf updated from the web-ui
    <unit id="bu5PltH" name="example">
      <notes>
        <note category="file-source" priority="1">app/templates/ex/show.html.twig:219</note>
        <note category="state" priority="1">new</note>
      </notes>
      <segment>
        <source>example</source>
        <target>{count, plural, =0 {There are no apples} one {There is one apple...} other {There are # apples!}}</target>
      </segment>
    </unit>
```

```twig
{{ "example"|trans({'%count%': 3}) }}
```

**Comment by bocharsky-bw** [comment]
Now can be closed thanks to #452
